content/docs/pipelines/sdk/component-development.md

```diff
@@ -79,12 +79,10 @@ from pathlib import Path
 
 # Function doing the actual work (Outputs first N lines from a text file)
 def do_work(input1_file, output1_file, param1):
-  for x in range(param1):
-    line = next(input1_file)
-    if not line:
-      break
+  for x, line in enumerate(input1_file):
+    if x >= param1:                                                                                                      break
```

Comment:
Looks like a newline was lost. Please move the break to the next line, indented two spaces past the "if"

```
    if x >= param1:
      break
```

Reply:
Thanks @joeliedtke 

Look like the break statement is lost when I merge. And I am afraid that I messed up my branch during merge. May need to revert and push again :-(

Reply:
Looks like I was able to update this. Thanks for working on this!   :)